uswds/uswds-site
# USWDS-Site - Feature: Add ECAP to Showcase

# Summary

Add [ECAP](https://nnsa.nsis.anl.gov/ecap/) to showcase.

> [!important]
> We need to update the changelog date before merge

## Related issue

Closes #2704 

## Preview link

[Showcase page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/brunerae-showcase-ecap/documentation/showcase/)

## Testing and review

1. Navigate to Showcase
2. Confirm Export Compliance Assistance Program is listed
3. Confirm the changelog is present and accurate
